manual/src/threelang/tfgrid/k8s.md

```diff
@@ -0,0 +1,66 @@
+# K8s Namespace
+
+- To deploy a kubernetes cluster, use the K8s namespace
+
+## Create Operation
+
+- action name: !!tfgrid.k8s.create
+- parameters:
+  - model_name [required]
+  - farm_id [optional]
+    - if 0, cluster nodes could span multiple nodes on different farms
+  - number_of_workers [required]
+    - a number in the range [1, 252]
+  - capacity [required]
+    - a string in ['small', 'medium', 'large'] indicating the capacity of each cluster node
+    - small: 2 vCPU, 4GB RAM, 10GB SSD
+    - medium: 4 vCPU, 8GB RAM, 20GB SSD
+    - large: 8 vCPU, 16GB RAM, 30GB SSD
+  - disk_size [optional]
+    - size in GB of disk to be mounted on each node at "/mnt/disk"
+  - ssh_key [required]
+- arguments:
+  - public_network
+  - master_public_ip
```

Comment:
What does this mean? That you want the master to have a public ip? If yes rename to add_public_ip_to_master